runtime/near-vm-logic/src/logic.rs

```diff
@@ -2434,6 +2434,7 @@ impl<'a> VMLogic<'a> {
         }
     }
 
+    // TODO: remove, as those costs are incorrectly computed, and we shall account it on deployment.
```

Comment:
can we create an issue (or link to an existing one) for this?

Reply:
https://github.com/near/nearcore/issues/4310